windows/tools/OobSimulator/MainWindow.xaml

```diff
@@ -0,0 +1,66 @@
+<!-- Copyright (c) Microsoft Corporation and Contributors. -->
+<!-- Licensed under the MIT License. -->
+
+<Window
+    x:Class="OobSimulator.MainWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:OobSimulator"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid>

```

Comment:
Have you considered some kind of top-level navigational element, like a [NavigationView](https://learn.microsoft.com/en-us/windows/apps/design/controls/navigationview)? If this app is ever going to have more than 1 feature (I'd guess yes), then it's important to plan for the overall navigation structure now. I'd lean towards adding one here than not.

Reply:
That's a good idea, I can add one

Reply:
As discussed offline, let's leave to separate PR.